packages/appstore/src/components/modals/real-wallets-upgrade/wallet-linking-step/wallet-linking-step.scss

```diff
@@ -0,0 +1,64 @@
+.wallet-linking-step {
+    width: 100%;
+    height: calc(57.4rem); // 73.4rem (modal-height) - 16rem (~15.2rem height of header + footer + margin from top)
```

Comment:
@aum-deriv It couldn't be responsive? 😕

Reply:
@farzin-deriv I tried making it responsive but my linking component was overflowing 😞.